Alignment/CommonAlignment/scripts/tkal_create_file_lists.py

```diff
@@ -363,14 +366,14 @@ def _request_dataset_information(self):
         for d in self._datasets: print_msg("\t"+d)
         print_msg("This may take a while...")
 
-        result = pool.map_async(get_events_per_dataset, self._datasets).get(sys.maxsize)
+        result = pool.map_async(get_events_per_dataset, self._datasets).get(3600)
```

Comment:
@smuzaffar ```sys.maxint``` has gone in python3, but isn't ```sys.maxsize``` still available, according to https://docs.python.org/3/library/sys.html ? What is the rationale behind this hardcoded value?

Reply:
ints are gone in python3. so maxsize is available but does not give the same answer as in python2. 

The error is
OverflowError: timestamp too large to convert to C _PyTime_t (which instead is a 64-bit int in C)

but perhaps 2147483647 would conserve the old behavior?

Reply:
@fabiocos , sys.maxint and sys.maxsize are used here to set a timeout (i.e. 9223372036854775807 sec).  There is no need to set such long timeouts.

Reply:
@davidlange6 according to the documentation ```sys.maxsize``` is now a variable of type ```Py_ssize_t``` so not what is desired here, I agree

Reply:
@smuzaffar this argument is more related to the functionality of the unit test I would say, waiting for an infinite time is not what is desired